lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -1,4 +1,5 @@
-import CampaignController from '../campaigns/Controller';
+import CampaignController from '../campaign/Controller';
```

Comment:
LOL. Yeah, I hate it when that kinda thing happens and you feel like you're taking crazy pills cause it looks right, but turns out it's a single character typo @_@